README.md

```diff
@@ -159,10 +159,19 @@ Here's a complete example of all supported config keys explained for the `contro
 ```yaml
 # Keys beginning with "cpln_" correspond to your settings in Control Plane.
 
+# Global settings that apply to `cpl` usage.
+# You can opt out of allowing the use of CPLN_ORG and CPLN_APP env vars
+# to avoid any accidents with the wrong org / app.
+allow_org_override_by_env: true
+allow_app_override_by_env: true
```

Comment:
1. allow opt out by app
2. default for production is to opt out in example, and default is off if not set
3. indicate that it's advisable for production to have this off

Reply:
Should we by default allow override or prevent it? This setup means we prevent it by default unless the user explicitly sets this option to `true`.

I think it should be opposite. We should allow by default and prevent it if it is explicitly declared.

I would suggest replacing `allow_x_override_by_env: true` to `prevent_x_overriden_by_env: false`. This way, we only prevent the override if such entry exist in the config file and it is set to `true`.